test/automl/test_multiclass.py

```diff
@@ -202,6 +202,7 @@ def test_custom_metric(self):
         print(automl_experiment.best_iteration)
         print(automl_experiment.best_estimator)
         automl_experiment = AutoML()
+        automl_experiment._state.metric = custom_metric
```

Comment:
why?

Reply:
I informed you in the teams chat and you didn’t respond. However, there is no way I can refer to the Teams chat here. I tried attaching a screenshot or a link both failed. Therefore please look for my chat yourself.